### ISSUE TITLE
clairctl generates empty reports with clair:v2.0.0-pre.4

### ISSUE THREAD

**Issue by jdel** [jgsqware/clairctl#17]
Yesterday I upgraded to clair:v2.0.0-pre.4 to try out the alpine support but clairctl generates empty reports. All my report that had vulnerabilities when running clair:v1.2.6 now show empty

analyze shows some vulnerabilities, but the report generated shows nothing.

```
~ # clairctl analyze registry.mydomain.net/core/custom-nginx

Image: registry.mydomain.net/registry.mydomain.net/core/custom-nginx:latest
 12 layers found

  ➜ Analysis [sha256:04e68] found 134 vulnerabilities.
  ➜ Analysis [sha256:a38d6] found 134 vulnerabilities.
  ➜ Analysis [sha256:b58c1] found 134 vulnerabilities.
  ➜ Analysis [sha256:f980b] found 134 vulnerabilities.
  ➜ Analysis [sha256:f788b] found 134 vulnerabilities.
  ➜ Analysis [sha256:762bf] found 134 vulnerabilities.
  ➜ Analysis [sha256:1f0d4] found 134 vulnerabilities.
  ➜ Analysis [sha256:97369] found 134 vulnerabilities.
  ➜ Analysis [sha256:6761f] found 134 vulnerabilities.
  ➜ Analysis [sha256:9de86] found 120 vulnerabilities.
  ➜ Analysis [sha256:31ad8] found 120 vulnerabilities.
  ➜ Analysis [sha256:43c26] found 36 vulnerabilities.
```

**Comment by jgsqware** [comment]
Thanks for reporting, I will have a look on that

**Comment by jgsqware** [comment]
Ok, the vulnerability type change between 1.2.6, and to 2.0.0. I will push a branch for version 2.0.0.

**Comment by jdel** [comment]
That would be great thanks !

**Comment by jdel** [comment]
To be closed, clairctl master works well with clair-2.0.0
